src/borg/archiver.py

```diff
@@ -3824,6 +3824,8 @@ def define_borg_mount(parser):
                                           formatter_class=argparse.RawDescriptionHelpFormatter,
                                           help='list archive or repository contents')
         subparser.set_defaults(func=self.do_list)
+        subparser.add_argument('--consider-checkpoints', action='store_true', dest='consider_checkpoints',
+                help='Show checkpoints in the repository contents. (Hidden by default)')
```

Comment:
Show checkpoint archives in the repository contents list (default: hidden).

Reply:
Will make this change.